doc/nrf/ug_thread_commissioning.rst

```diff
@@ -52,10 +52,35 @@ For details about scenarios that include devices with multiple roles, see `Threa
 
 See :ref:`thread_ot_commissioning_cli` for the list of commands used to assign the commissioning roles.
 
+.. _thread_ot_device_types:
+
+Device types
+============
+
+OpenThread distinguishes between different device types depending on what roles a device can take in the Thread network.
+
+Full Thread Device (FTD)
+  An FTD can be both a router and an end device.
+  Its radio is always on.
+
+Minimal Thread Device (MTD)
+  An MTD is always an end device.
+  It forwards all messages to its parent.
+
+  There are two important subtypes:
+
+  Minimal End Device (MED)
+    An MED keeps its transceiver always on.
+
+  Sleepy End Device (SED)
+   An SED is usually off and wakes occasionally to poll for messages from its parent.
```

Comment:
@edmont, @MarekPorwisz, should we rephrase it taking the CSL into account?

Reply:
CSL is just a feature SEDs can use at any time. But yes, maybe something a bit more generalistic could be stated, like:

_An SED is usually off and wakes occasionally to receive messages from its parent._